scripts/docker_run.bat

```diff
@@ -0,0 +1,2 @@
+@echo  [+] (Win_OS) Running Docker Container.
+docker container run --name devi -d -p 8501:8501 bennywestsyde/gatorminer && echo "\n\tYou can now view your Streamlit app in your browser.\n\n\tNetwork URL: http://localhost:8501 \n\tExternal URL: http://141.195.4.17:8501"
```

Comment:
Again, this is using @BennyWestsyde docker image, please make sure to change this once the docker image is published to the Allegheny CS Department docker hub organization